backend/app/utils/FfMpeg.scala

```diff
@@ -0,0 +1,24 @@
+package utils
+
+import java.nio.file.Path
+import scala.sys.process.{Process, ProcessLogger, stdout}
+
+object FfMpeg extends Logging {
+
+  case class FfMpegSubprocessCrashedException(exitCode: Int, stderr: String) extends Exception(s"Exit code: $exitCode: ${stderr}")
+
+  def convertToWav(originalFile: Path, tmpDir: Path): Path = {
+    val ffMpegStdErrLogger = new BasicStdErrLogger()
+    val tempFile = tmpDir.resolve(s"${originalFile.getFileName}.wav")
+    val cmd = s"ffmpeg -i $originalFile -ar 16000 -ac 1 -c:a pcm_s16le ${tempFile}"
```

Comment:
Do we need to think about how to install `ffmpeg` in dev and prod environments? Or did we say it comes with whisper 🤔

Reply:
I think it would be worth adding here https://github.com/guardian/giant/blob/main/Brewfile, I have already added it here https://amigo.gutools.co.uk/recipes/investigations-giant-app-java11

Reply:
Thanks for the comments, updated the brew file 👍